src/controller_pak.h

```diff
@@ -19,18 +19,18 @@
 
 #define COMPANY_CODE 0x3459
 
-enum SIDeviceStatus {
+typedef enum {
     CONTROLLER_PAK_GOOD,
-	NO_CONTROLLER_PAK,
+    NO_CONTROLLER_PAK,
     CONTROLLER_PAK_INCONSISTENT,
-	CONTROLLER_PAK_WITH_BAD_ID,
-	CONTROLLER_PAK_FULL,
-	CONTROLLER_PAK_CHANGED, // Also used when there's no free space
-	UNK6,
-	RUMBLE_PAK,
-	UNK8, // func_80074B34 Sets this
-	CONTROLLER_PAK_BAD_DATA,
-};
+    CONTROLLER_PAK_WITH_BAD_ID,
+    CONTROLLER_PAK_FULL,
+    CONTROLLER_PAK_CHANGED,
+    UNK6, //Could be another way to mark as full according to tt_menu_loop
+    RUMBLE_PAK,
+    UNK8, // func_80074B34 Sets this
```

Comment:
```suggestion
    CONTROLLER_PAK_UNK6, //Could be another way to mark as full according to tt_menu_loop
    RUMBLE_PAK,
    CONTROLLER_PAK_UNK8, // func_80074B34 Sets this
```
Just because there's no namespacing or enum classes, we should avoid generic enum values.